fe/fe-core/src/main/java/com/starrocks/catalog/FunctionSet.java

```diff
@@ -48,177 +48,330 @@ public class FunctionSet {
+    public static final String DEGRESS = "degress";
+    public static final String DIVIDE = "divide";
+    public static final String E = "e";
+    public static final String EXP = "exp";
+    public static final String DEXP = "dexp";
+    public static final String FLOOR = "floor";
+    public static final String DFLOOR = "dfloor";
+    public static final String FMOD = "fmod";
+    public static final String GREATEST = "greatest";
+    public static final String LEAST = "least";
+    public static final String LN = "ln";
+    public static final String DLOG1 = "dlog1";
+    public static final String LOG = "log";
+    public static final String LOG2 = "log2";
+    public static final String LOG10 = "log10";
+    public static final String DLOG10 = "dlog10";
+    public static final String MOD = "mod";
+    public static final String MULTIPLY = "multiply";
+    public static final String NEGATIVE = "negative";
+    public static final String PI = "pi";
+    public static final String PMOD = "pmod";
+    public static final String POSITIVE = "positive";
+    public static final String POW = "pow";
+    public static final String POWER = "power";
+    public static final String DPOW = "dpow";
+    public static final String FPOW = "fpow";
+    public static final String RADIANS = "radians";
+    public static final String RAND = "rand";
+    public static final String RANDOM = "random";
+    public static final String ROUND = "round";
+    public static final String DROUND = "dround";
+    public static final String SIGN = "sign";
+    public static final String SIN = "sin";
+    public static final String SQRT = "sqrt";
+    public static final String SUBTRACT = "subtract";
+    public static final String DSQRT = "dsqrt";
+    public static final String SQUARE = "square";
+    public static final String TAN = "tan";
+    public static final String TRUNCATE = "truncate";
+
+    // Window functions:
+    public static final String LEAD = "lead";
+    public static final String LAG = "lag";
+    public static final String FIRST_VALUE = "first_value";
+    public static final String FIRST_VALUE_REWRITE = "first_value_rewrite";
+    public static final String LAST_VALUE = "last_value";
+    public static final String DENSE_RANK = "dense_rank";
+    public static final String RANK = "rank";
+    public static final String NTILE = "ntile";
+    public static final String ROW_NUMBER = "row_number";
+
+    // Other functions:
+    public static final String HLL_HASH = "hll_hash";
+    public static final String HLL_CARDINALITY = "hll_cardinality";
+    public static final String DEFAULT_VALUE = "default_value";
+    public static final String REPLACE_VALUE = "replace_value";
+
+    // JSON functions
+    public static final Function JSON_QUERY_FUNC = new Function(
+            new FunctionName(JSON_QUERY), new Type[] {Type.JSON, Type.VARCHAR}, Type.JSON, false);
+
+    private static final Logger LOGGER = LogManager.getLogger(FunctionSet.class);
```

Comment:
other classes are all `LOG`, needn't change this name.

Reply:
emm, the name is occupied by `public static final String LOG = "log;`, so I changed it to `LOGGER`